lib/datawow/models/moderations/moderation.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Datawow
+  # :nodoc:
+  class Moderation
+    include Datawow::Models::Interface
+
+    attr_writer :project_key
+
+    def initialize(token = nil)
```

Comment:
`def initialize(token)`
should it raise error when no token

Reply:
param token is not needed since it can be set via `Datawow.project_key`

it can be used like this
```
Datawow.project_key = 'key'
Datawow.moderation.create(...)
```
or
```
Datawow::Moderation.new('key').create(...)
```